tests/test_core.py

```diff
@@ -199,182 +199,194 @@ def test_insert_tables(engine_testaccount):
+                assert (
+                    len([row for row in results]) == 4
+                ), "number of rows from addresses table"
+                results.close()
+
+                # select specified column names
+                results = conn.execute(
+                    select(users.c.name, users.c.fullname).order_by("name")
                 )
-            )
-            == "users JOIN addresses "
-            "ON addresses.email_address LIKE users.name || :name_1"
-        )
+                results.fetchone()
+                row = results.fetchone()
+                assert row._mapping["name"] == "wendy", "name"
 
-        s = select(users.c.fullname).select_from(
-            users.join(addresses, addresses.c.email_address.like(users.c.name + "%"))
-        )
-        results = conn.execute(s).fetchall()
-        assert results[1] == ("Jack Jones",)
+                # join
+                results = conn.execute(
+                    select(users, addresses).where(users.c.id == addresses.c.user_id)
+                )
+                results.fetchone()
+                results.fetchone()
+                results.fetchone()
+                row = results.fetchone()
+                assert row._mapping["email_address"] == "wendy@aol.com", "email address"
 
-        s = (
-            select(users.c.fullname)
-            .select_from(users.outerjoin(addresses))
-            .order_by(users.c.fullname)
-        )
-        results = conn.execute(s).fetchall()
-        assert results[-1] == ("Wendy Williams",)
-    finally:
-        conn.close()
-        # drop tables
-        addresses.drop(engine_testaccount)
-        users.drop(engine_testaccount)
+                # Operator
+                assert (
+                    str(users.c.id == addresses.c.user_id)
+                    == "users.id = addresses.user_id"
+                ), "equal operator"
+                assert (
+                    str(users.c.id == 7) == "users.id = :id_1"
+                ), "equal to a static number"
+                assert str(users.c.name == None)  # NOQA
+                assert (
+                    str(users.c.id + addresses.c.id) == "users.id + addresses.id"
+                ), "number + number"
+                assert (
+                    str(users.c.name + users.c.fullname)
+                    == "users.name || users.fullname"
+                ), "str + str"
+
+                # Conjunctions
+                # example 1
+                obj = and_(
+                    users.c.name.like("j%"),
+                    users.c.id == addresses.c.user_id,
+                    or_(
+                        addresses.c.email_address == "wendy@aol.com",
+                        addresses.c.email_address == "jack@yahoo.com",
+                    ),
+                    not_(users.c.id > 5),
+                )
+                expected_sql = textwrap.dedent(
+                    """\
+                users.name LIKE :name_1
```

Comment:
```suggestion
                 users.name LIKE :name_1
```

Reply:
actually we need the extra space such that precommit won't strip the space for us..